web/pgadmin/static/js/helpers/Notifier.jsx

```diff
@@ -175,9 +175,27 @@ class Notifier {
     this.modal.alert(title, text, onOkClick, okLabel);
   }
 
-  confirm(title, text, onOkClick, onCancelClick, okLabel=gettext('Yes'), cancelLabel=gettext('No'), extras=null) {
+  confirm(title, text, onOkClick, onCancelClick, okLabel=gettext('Yes'), cancelLabel=gettext('No'), extras=null, type=null) {
```

Comment:
You're adding more parameters which will eventually make SonarQube angry. I feel confirm should take only first 5 args. For custom confirm, pass and object to the confirm function. So basically confirm should be flexible to accept both types of args.